testsuite/run_sets/secondary_parallelizable.yml

```
@@ -37,6 +37,7 @@
 - features/secondary/srv_payg_ssh_connection.feature
 - features/secondary/srv_push_package.feature
 - features/secondary/srv_reportdb.feature
+- features/secondary/srv_check_dist_channel_mapping_page.feature
```

We can remove some words from the feature file :)

```suggestion
- features/secondary/srv_dist_channel_mapping.feature
```